samples/msal-node-samples/ElectronSystemBrowserTestApp/src/Main.ts

```diff
@@ -0,0 +1,209 @@
+            : authConfig;
+    }
+
+    private static setDefaultProtocol(): void {
+        if (process.defaultApp) {
+            if (process.argv.length >= 2) {
+                Main.application.setAsDefaultProtocolClient(
+                    authConfig.customProtocol.name,
+                    process.execPath,
+                    [path.resolve(process.argv[1])]
+                );
+            } else {
+                Main.application.setAsDefaultProtocolClient(
+                    authConfig.customProtocol.name
+                );
+            }
+        }
+    }
+
+    private static requestSingleInstance(): void {
+        // Prevent private URI scheme notifications on Windows + Linux from creating a new instance of the application
+        const gotTheLock = app.requestSingleInstanceLock();
+        if (!gotTheLock) {
+            Main.application.quit();
+        } else {
+            Main.application.on("second-instance", Main.onSecondInstance);
+            Main.application.on("ready", Main.onReady);
+            Main.application.on("open-url", Main.onOpenUrl);
+        }
+    }
+
+    private static async loadBaseUI(): Promise<void> {
+        await Main.mainWindow.loadFile(path.join(__dirname, "../index.html"));
+    }
+
+    private static onWindowAllClosed(): void {
+        // Windows and Linux will quit the application when all windows are closed
+        if (process.platform !== "darwin") {
+            // macOS requires explicit quitting
+            Main.application.quit();
+        }
+    }
+
+    private static onClose(): void {
+        Main.mainWindow = null;
+    }
+
+    /**
+     * This method should focus on our window when running a Second instance of the app
+     */
+    private static handleWindowState(): void {
+        // Someone tried to run a second instance, we should focus our window.
+        if (Main.mainWindow) {
+            if (Main.mainWindow.isMinimized()) Main.mainWindow.restore();
+            Main.mainWindow.focus();
+        }
+    }
+
+    /**
+     * On Windows and Linux, this is where we receive login responses
+     */
+    private static onSecondInstance(event: any, commandLine: string[]): void {
+        event.preventDefault();
+        Main.handleWindowState();
+        const url = Main.getDeepLinkUrl(commandLine);
+        if (url) Main.getCodeFromDeepLinkUrl(url);
+    }
+
+    /**
+     * On macOS, this is where we receive login responses
+     */
+    private static onOpenUrl(event: any, schemeData: string): void {
+        event.preventDefault();
+        Main.handleWindowState();
+        Main.getCodeFromDeepLinkUrl(schemeData);
+    }
+
+    /**
+     * Get the deep Linked from command line params
+     */
+    private static getDeepLinkUrl(argv: string[]): string {
+        for (const arg of argv) {
+            const value = arg;
+            if (value.indexOf(authConfig.customProtocol.name) !== -1) {
+                return value;
+            }
```

Comment:
So here's another example where you didn't put this on one line, even though you did so on line 83. I would say go through this entire review and make it all consistent.